test/pkgs/Dynamo Samples/extra/CustomRenderExample.dyn

```diff
@@ -0,0 +1,84 @@
+{
+  "Uuid": "6b11e76c-b00c-4300-a7a6-44b5e369c268",
+  "IsCustomNode": false,
+  "Description": null,
+  "Name": "CustomRenderExample",
+  "ElementResolver": {
+    "ResolutionMap": {}
+  },
+  "Inputs": [],
+  "Outputs": [],
+  "Nodes": [
+    {
+      "ConcreteType": "Dynamo.Graph.Nodes.ZeroTouch.DSFunction, DynamoCore",
+      "NodeType": "FunctionNode",
+      "FunctionSignature": "Examples.CustomRenderExample.Create",
+      "Id": "a0a49dec368841ef9f140858fef31188",
+      "Inputs": [],
+      "Outputs": [
+        {
+          "Id": "e751ff6164b4415ca6561fca14fbfa9a",
+          "Name": "CustomRenderExample",
+          "Description": "CustomRenderExample",
+          "UsingDefaultValue": false,
+          "Level": 2,
+          "UseLevels": false,
+          "KeepListStructure": false
+        }
+      ],
+      "Replication": "Auto",
+      "Description": "CustomRenderExample.Create ( ): CustomRenderExample"
+    }
+  ],
+  "Connectors": [],
+  "Dependencies": [],
+  "NodeLibraryDependencies": [
+    {
+      "Name": "Dynamo Samples",
+      "Version": "2.0.1",
```

Comment:
Notice the graph is serialized with a higher version, so opening this graph in test will display dependencyViewExtension so we can test the functionality there